lib/widgets/pop_up_join_chat.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:flutter/material.dart';
+import 'package:hobihub/screens/chat_screen.dart';
+import 'package:hobihub/screens/home_screen.dart';
+
+class JoinGroupConfirmation extends StatelessWidget {
+  final String title;
+
+  const JoinGroupConfirmation({Key? key, required this.title})
```

Comment:
ini juga samain ke awal...karena kepanjangan @ketrinvania

Reply:
ini udah paling awal jir wkwkw

Reply:
const JoinGroupConfirmation({super.key, required this.title});

gua ubah jadi kek gini ya?